src/app/authorization/decline-invitation/decline-invitation.component.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../../../public/assets/styles/variables' as *;
+@use '../../../../public/assets/styles/reset' as *;
+
+.declined-title{
```

Comment:
We can use the `nesting `concept here to avoid directly selecting class names by creating a nested structure like:

```
.declined {  
  .declined-desc {  
    // styles for declined-desc  
  }  
  .declined-title {  
    // styles for declined-title  
  }  
}
```